### ISSUE TITLE
Error: cannot import name 'LayerNormalization' from 'tensorflow.python.keras.layers.normalization' 

### ISSUE THREAD

**Issue by OliveS9** [tensorflow/tensorflow#54401]
How to fix the following error?

`Error: cannot import name 'LayerNormalization' from 'tensorflow.python.keras.layers.normalization' (/usr/local/lib/python3.7/dist-packages/tensorflow/python/keras/layers/normalization/__init__.py)`

Thanks!

**Comment by mohantym** [comment]
Hi @OliveS9 ! Could you please fill the template which will  help us expedite the issue? You can try following options though.
1. use tf.keras instead of keras 
2. use tf.keras.layers.LayerNormalization  
2. uninstall and reinstall latest tensorflow version in a new environment.

**Comment by OliveS9** [comment]
@mohantym Thank you for your reply, but how to use "tf.keras"? Thanks.

**Comment by mohantym** [comment]
Just replace this command
`import keras `
with
`from tensorflow import keras `
